src/integrations/walletIntegration.ts

```diff
@@ -18,6 +18,11 @@ export default async function walletIntegration(store: Store<any>) {
     registerProvider: dispatchProviderAction(registerProvider),
   }
 
+  // TODO: fetch approvedTokens list from api
+  // then after getting tokensJSON in getDefaultTokens create a list of approved TokenCodes
+  // then only dispatch that list
+  dispatch(setApprovedTokens(['ETH', 'GNO']))
```

Comment:
this is only `dispatch`ing the `symbols` yet you check the whole `TokenObject` in `isTokenApproved`

Reply:
Why is it here? This is old, I don't have it locally